libs/components/modals/src/lib/modules/modal/modal.component.scss

```diff
@@ -70,6 +77,20 @@
     outline-width: thin;
     outline-offset: -1px;
   }
+
+  &:has(> .sky-layout-host-fit) {
+    flex-grow: 1;
+    height: var(--sky-modal-content-max-height, 100%);
+    position: relative;
+  }
+
+  .sky-layout-host-fit ::ng-deep sky-modal-content {
```

Comment:
Can this move to the modal content component's stylesheet using `:host-context()` in order to avoid `::ng-deep` here?

```
:host-context(sky-modal .sky-layout-host-fit) {
  :host {
    padding: 0;
  }
}
```